README.md

```diff
@@ -6,7 +6,7 @@ Easily organize windows without using a mouse.
 
 ## System requirements
 
-Spectacle [0.8.6][1] is the last version to support Mac OS X 10.7; future releases will only support 10.9 or greater. Folks wishing to stay on 10.7 should download [0.8.6][1]. Those looking for a version of Spectacle that supports 10.6 can still download [0.7][2].
+Spectacle [0.8.6][1] is the last version to support macOS 10.7; future releases will only support 10.9 or greater. Folks wishing to stay on 10.7 should download [0.8.6][1]. Those looking for a version of Spectacle that supports 10.6 can still download [0.7][2].
```

Comment:
This reference is not accurate.  OS versions of 10.12 and up are named Mac OS, 10.11 and lower would be more accurate to reference as OS X.  Perhaps there is another way to reference the difference for future users.